ui/page/root/home_page.go

```diff
@@ -405,6 +450,109 @@ func (hp *HomePage) notificationSettingsLayout(gtx C) D {
 	)
 }
 
+func (hp *HomePage) startSyncing(wallet sharedW.Asset, unlock load.NeedUnlockRestore) {
+	// Watchonly wallets do not have any password neithers need one.
+	if !wallet.ContainsDiscoveredAccounts() && wallet.IsLocked() && !wallet.IsWatchingOnlyWallet() {
+		hp.unlockWalletForSyncing(wallet, unlock)
+		return
+	}
+	unlock(true)
+
+	if hp.isConnected {
+		// once network connection has been established proceed to
+		// start the wallet sync.
+		if err := wallet.SpvSync(); err != nil {
+			log.Debugf("Error starting sync: %v", err)
+		}
+		return
```

Comment:
Missed this. I think we should fall through (i.e remove the `return`) so we get to try again after the first 10 sec until we successfully start syncing.